PIPS/pip-008.md

```diff
@@ -0,0 +1,28 @@
+    PIP: 008
+    Title: On-A-Budget mode (aka Doshirak mode)
+    Author: <Kosta Korenkov>
+    Created: <2018-03-18>
+    Affects: PIP004, PIP005, PIP007
+    Status: draft
+
+
+## Simple Summary
+Cut our costs to bare minimum while we are looking for extra funding. This PIP is basically a bunch of amendments to PIP004, PIP005, PIP007 which are activated conditionally.
+
+## Motivation
+We are running out of cash and our top priority should be to look for investors/grants and that's what we should spent most of the money for.
+
+## Activation/Deactivation
+This PIP comes in effect on Monday 00:00 UTC if Escrow council contains less than 50,000 DAI worth of ETH.
+This PIP is deactivated once Escrow council contains more than 80,000 DAI worth on ETH.
+
+## Specification
+- PIP004: Limit partner buyback program to buy back 60,000 PSC maximum
+- PIP007: Exclude Acebusters circle from Budget program
+- PIP007: Limit budget for Communication circle to 2000 DAI per month
+- PIP007: budgets can only be spent to reimburse expenses. If you want to earn extra money, create a bounty ([PIP005](https://github.com/parsec-labs/PIPs/blob/master/PIPS/pip-005.md)).
+- PIP005: Exclude Acebusters circle from Bounty program
+- PIP005: Bounties bigger than 1000 DAI can be claimed only if more money attracted to ParsecLabs as part of the bounty (investors/grants) 
```

Comment:
how to prove that?

Reply:
specify transaction which funds Escrow? I agree, it is super fuzzy. I have some suggestions — will write on Slack